### PR TITLE
alter metadata.user

### PR DESCRIPTION
**Pull request by AllinChen**

### CODE REVIEW

File: api/v1/metadata/user.go [hunk 1]
```diff
@@ -623,12 +659,12 @@ func GetMiddlewareClustersByUserID(c *gin.Context) {
 }
 
 // @Tags 	user
-// @Summary get mysqlclusterclusters by id
+// @Summary get mysqlclusters by id
 // @Accept	application/json
 // @Param	id path int true "user id"
-// @Produce mysqlclustercluster/json
+// @Produce application/json
 // @Success 200 {string} string "{"mysql_clusters": [{"id": 1,"cluster_name": "mysql-cluster-pmm2","middleware_cluster_id": 0,"env_id": 1"monitor_system_id": 1,"del_flag": 0,"last_update_time": "2021-12-21T09:16:10.750725+08:00","create_time": "2021-09-02T09:02:22.346672+08:00",},]}"
```
Comment (romberli) [inline api/v1/metadata/user.go:666]: the return value should be middleware clusters

File: api/v1/metadata/user.go [hunk 2]
```diff
@@ -623,12 +659,12 @@ func GetMiddlewareClustersByUserID(c *gin.Context) {
...
-// @Router 	/api/v1/metadata/user/mysqlclustercluster/:id [get]
+// @Router 	/api/v1/metadata/user/mysqlcluster/:id [get]
```
Comment (romberli) [inline api/v1/metadata/user.go:667]: the path should be middleware-cluster

File: internal/app/metadata/user_repository.go [hunk 3]
```diff
@@ -489,3 +520,91 @@ func (ur *UserRepo) GetMySQLClustersByUserID(userID int) ([]metadata.MySQLCluste
...
+
+	result, err := ur.Execute(sql, id, id, id)
+	if err != nil {
+		return nil, err
+	}
+
+	resultNum := result.RowNumber()
+	mysqlserverList := make([]metadata.MySQLServer, resultNum)
+
+	for row := 0; row < resultNum; row++ {
```
Comment (romberli) [inline internal/app/metadata/user_repository.go:600]: use constant.ZeroInt instead of 0

File: pkg/message/metadata/user.go [hunk 4]
```diff
@@ -60,58 +64,66 @@ const (
...
-	message.Messages[DebugMetadataGetDBsByUserID] = config.NewErrMessage(message.DefaultMessageHeader, DebugMetadataGetDBsByUserID, "metadata: get db id list completed. message: %s")
-	message.Messages[DebugMetadataGetMiddlewareClustersByUserID] = config.NewErrMessage(message.DefaultMessageHeader, DebugMetadataGetMiddlewareClustersByUserID, "metadata: get middlewarecluster id list completed. message: %s")
-	message.Messages[DebugMetadataGetMySQLClustersByUserID] = config.NewErrMessage(message.DefaultMessageHeader, DebugMetadataGetMySQLClustersByUserID, "metadata: get mysqlcluster id list completed. message: %s")
+	message.Messages[DebugMetadataGetAppsByUserID] = config.NewErrMessage(message.DefaultMessageHeader, DebugMetadataGetAppsByUserID, "metadata: get app list completed. message: %s")
+	message.Messages[DebugMetadataGetDBsByUserID] = config.NewErrMessage(message.DefaultMessageHeader, DebugMetadataGetDBsByUserID, "metadata: get db list completed. message: %s")
+	message.Messages[DebugMetadataGetMiddlewareClustersByUserID] = config.NewErrMessage(message.DefaultMessageHeader, DebugMetadataGetMiddlewareClustersByUserID, "metadata: get middlewarecluster list completed. message: %s")
```
Comment (romberli) [inline pkg/message/metadata/user.go:86]: replace middlewarecluster with middleware cluster, same for the following mysqlcluster or mysqlserver

File: pkg/message/metadata/user.go [hunk 5]
```diff
@@ -60,58 +64,66 @@ const (
...
-	message.Messages[InfoMetadataGetAccountName] = config.NewErrMessage(message.DefaultMessageHeader, InfoMetadataGetAccountName, "metadata: get user by accountname completed.accountName: %s")
-	message.Messages[InfoMetadataGetEmail] = config.NewErrMessage(message.DefaultMessageHeader, InfoMetadataGetEmail, "metadata: get user by email completed.email: %s")
-	message.Messages[InfoMetadataGetTelephone] = config.NewErrMessage(message.DefaultMessageHeader, InfoMetadataGetTelephone, "metadata: get user by telephone completed.telephone: %s")
+	message.Messages[InfoMetadataGetByUserName] = config.NewErrMessage(message.DefaultMessageHeader, InfoMetadataGetByUserName, "metadata: get user by username completed. Name: %s")
+	message.Messages[InfoMetadataGetEmployeeID] = config.NewErrMessage(message.DefaultMessageHeader, InfoMetadataGetEmployeeID, "metadata: get user by employeeid completed. employID: %d")
+	message.Messages[InfoMetadataGetAccountName] = config.NewErrMessage(message.DefaultMessageHeader, InfoMetadataGetAccountName, "metadata: get user by accountname completed. accountName: %s")
```
Comment (romberli) [inline pkg/message/metadata/user.go:98]: replace accountname with account name

File: router/metadata.go [hunk 6]
```diff
@@ -117,5 +118,6 @@ func RegisterMetadata(group *gin.RouterGroup) {
 		metadataGroup.GET("/user/db/:id", metadata.GetDBsByUserID)
 		metadataGroup.GET("/user/middlewarecluster/:id", metadata.GetMiddlewareClustersByUserID)
 		metadataGroup.GET("/user/mysqlcluster/:id", metadata.GetMySQLClustersByUserID)
```
Comment (romberli) [inline router/metadata.go:120]: replace mysqlcluster with mysql-cluster

File: api/v1/metadata/user.go [hunk 7]
```diff
@@ -623,12 +659,12 @@ func GetMiddlewareClustersByUserID(c *gin.Context) {
 }
 
 // @Tags 	user
-// @Summary get mysqlclusterclusters by id
+// @Summary get mysqlclusters by id
```
[the remark this answers is not in the document]
Comment (AllinChen) [reply api/v1/metadata/user.go:662]: done

File: api/v1/metadata/user.go (hunk 1 above)
Comment (AllinChen) [reply api/v1/metadata/user.go:666]: done

File: api/v1/metadata/user.go (hunk 2 above)
Comment (AllinChen) [reply api/v1/metadata/user.go:667]: done

File: internal/app/metadata/user_repository.go (hunk 3 above)
Comment (AllinChen) [reply internal/app/metadata/user_repository.go]: done

File: internal/app/metadata/user_repository_test.go [hunk 8]
```diff
@@ -164,6 +167,18 @@ func TestUserRepo_GetByAccountName(t *testing.T) {
...
 
+func TestUserRepo_GetByAccountNameOrEmployeeID(t *testing.T) {
+	asst := assert.New(t)
+
+	entity, err := testUserRepo.GetByAccountNameOrEmployeeID(testUserAccountName)
+	asst.Nil(err, common.CombineMessageWithError("test GetByAccountNameOrEmployeeID failed", err))
+	asst.Equal(testUserAccountName, entity.GetAccountName(), "test GetByAccountNameOrEmployeeID failed")
+
+	entity, err = testUserRepo.GetByAccountNameOrEmployeeID(testUserEmployeeID)
+	asst.Nil(err, common.CombineMessageWithError("test GetByAccountNameOrEmployeeID failed", err))
```
[the remark this answers is not in the document]
Comment (AllinChen) [reply internal/app/metadata/user_repository_test.go]: done

File: internal/app/metadata/user_repository_test.go [hunk 9]
```diff
@@ -164,6 +167,18 @@ func TestUserRepo_GetByAccountName(t *testing.T) {
...
+	asst.Equal(testUserAccountName, entity.GetAccountName(), "test GetByAccountNameOrEmployeeID failed")
```
[the remark this answers is not in the document]
Comment (AllinChen) [reply internal/app/metadata/user_repository_test.go]: done

File: pkg/message/metadata/user.go (hunk 4 above)
Comment (AllinChen) [reply pkg/message/metadata/user.go]: done

File: pkg/message/metadata/user.go (hunk 5 above)
Comment (AllinChen) [reply pkg/message/metadata/user.go]: done

File: router/metadata.go (hunk 6 above)
Comment (AllinChen) [reply router/metadata.go]: done

File: api/v1/metadata/user.go [hunk 10]
```diff
@@ -699,7 +699,7 @@ func GetMySQLClustersByUserID(c *gin.Context) {
 }
 
 // @Tags 	user
-// @Summary get MySQLClusters by id
+// @Summary get all Mysql servers by id
```
Comment (romberli) [inline api/v1/metadata/user.go:702]: use lower case of mysql

**Review by romberli — CHANGES_REQUESTED** [review]

File: api/v1/metadata/user.go (hunk 10 above)
Comment (AllinChen) [reply api/v1/metadata/user.go]: done

**Review by romberli — APPROVED** [review]

**Pull request merged**